migrate.py

```diff
@@ -7,5 +7,12 @@
     dump = subprocess.check_output(
         "pg_dump --schema-only --no-owner --no-privileges f1", shell=True
     )
+    schema_migrations = db.fetchall(
+        "SELECT version FROM schema_migrations ORDER BY version ASC"
+    )
     with open("./db/structure.sql", mode="wb") as f:
         f.write(dump)
+
+        template = "INSERT INTO schema_migrations (version) VALUES (%s);\n"
+        for migration in schema_migrations:
+            f.write(str.encode(template % migration))
```

Comment:
Added a bit here to dump the migrations that have been run. This is pretty crucial to make good use of the structure file.

If you want to alter a migration before it has been run (i.e. it's sitting in a PR), you can do the following:
- `git checkout master db/structure.sql` - get the canonical copy of structure.sql
- `dropdb f1 && createdb f1 && cat db/structure.sql | psql f1` - clear the database and load the canonical structure
- `python migrate.py` - run the migrations

Rails has a nice shorthand for this: `rake db:drop db:create db:structure:load db:migrate` but we have a (for now) hacky equivalent.